ci/run-check.sh

```diff
@@ -45,19 +45,35 @@ case $TARGET in
     ;;
 
   try-runtime)
-    cargo build -p centrifuge-chain --release --features try-runtime
+    echo "Contents of target/release/wbuild/:"
+    ls -l target/release/wbuild/
+    echo "Contents of subdirectories in target/release/wbuild/:"
+    ls -l target/release/wbuild/*/
+    # Check if try-runtime is available
+    if ! command try-runtime --version &> /dev/null
+    then
+        echo "try-runtime could not be found, trying 'cargo install try-runtime-cli'"
+        curl -sL https://github.com/paritytech/try-runtime-cli/releases/download/v0.6.1/try-runtime-x86_64-unknown-linux-musl -o try-runtime
```

Comment:
Latest version is v0.7.0 which I have been using locally
```suggestion
        curl -sL https://github.com/paritytech/try-runtime-cli/releases/download/v0.7.0/try-runtime-x86_64-unknown-linux-musl -o try-runtime
```